xml/installation-installation-preinstall_checklist.xml

```diff
@@ -618,6 +618,7 @@
      <row>
       <entry>
 <screen>
+Enter the information for your system.
```

Comment:
`<?dbsuse-disable-trimming?>` needs to be the first non-space text in the screen to have any effect. Also, why are you adding the text "Enter the information..." to the screen directly, not in a para above it?